src/s1reader/s1_burst_slc.py

```diff
@@ -148,6 +148,7 @@ class Sentinel1BurstSlc:
     polarization: str # {VV, VH, HH, HV}
     burst_id: str # t{track_number}_{burst_index}_iw{1,2,3}
     platform_id: str # S1{A,B}
+    file_id: str # SAFE file name
```

Comment:
I find the name of the variable to be confusing. Shall we call it directly `safe_filename`. This would avoid misunderstandings

Reply:
Done.